src/main/java/dev/snowdrop/release/model/cpaas/release/SendWhenEnum.java

```diff
@@ -0,0 +1,25 @@
+package dev.snowdrop.release.model.cpaas.release;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+public enum SendWhenEnum {
```

Comment:
Is this enum used?

Reply:
Yes, in the `dev.snowdrop.release.model.cpaas.release.CPaaSReleaseTeam` class.